Index: chirpstack-udp-forwarder-4.1.8/Cargo.toml
===================================================================
--- chirpstack-udp-forwarder-4.1.8.orig/Cargo.toml
+++ chirpstack-udp-forwarder-4.1.8/Cargo.toml
@@ -30,6 +30,6 @@ chrono = "0.4"
   base64 = "0.22"
   prost = "0.13"
   prost-types = "0.13"
-  prometheus-client = "0.22"
+  prometheus-client = { git = "https://github.com/brocaar/client_rust.git", branch = "fix_issue_172" }
   lazy_static = "1.5"
   anyhow = "1.0"
